src/components/pages/Chart.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState, useRef, useEffect } from 'react';
+
+import { Container, Button  } from '@mui/material'
+
+import ChartJSWrapper from "chartjs-wrapper"
+
```

Comment:
I would probably name the npm module to something that includes opensim e.g. opensim-chart(js) or opensim-web-tools if not too hard. We'll likely incorporate the visualization later and maybe other pieces downstream

Reply:
I've been looking for ways of renaming it in npmjs and looks like it is not possible. I think, once I upload the code to opensim-org, we can call the repository opensim-web-tools and either have a package named opensim-web-tools that includes all of the react tools we create, or separate each component/functionality into different packages and call this one opensim-chart(js). What are your thoughts?

**Note:** The currently existing package cannot be removed from npm, but only deprecated. I think the reason is that it has been downloaded 2,296 times since I published it, so I guess deprecating it could cause problems to the people that downloaded it. I didn't expect so many people would download it in just a week.

Reply:
Thanks @AlbertoCasasOrtiz your package is very popular 🎆 🍾